CRM/Contact/Form/Contact.php

```diff
@@ -213,6 +213,9 @@ public function preProcess() {
         }
 
         $displayName = CRM_Contact_BAO_Contact::displayName($this->_contactId);
+        if ($defaults['is_deceased']) {
+          $displayName .= '  <font color=red>(deceased)</font>';
```

Comment:
`font` and `color` are old deprecated html tags. Please use css.
I'd recommend a `span` with a semantic css class like `crm-contact-deceased`